.travis.yml

```diff
@@ -5,12 +5,28 @@ matrix:
     # Standard builds
     - os: linux
       compiler: gcc
+      env: CONFIG=Debug
+    - os: linux
+      compiler: gcc
+      env: CONFIG=Release
+    - os: linux
+      compiler: gcc
+      env: CONFIG=RelWithDebInfo
     - os: linux
       compiler: clang
+      env: CONFIG=Release
+    - os: osx
+      compiler: gcc
+      env: CONFIG=Debug
+    - os: osx
+      compiler: gcc
+      env: CONFIG=Release
     - os: osx
       compiler: gcc
+      env: CONFIG=RelWithDebInfo
     - os: osx
       compiler: clang
+      env: CONFIG=Release
 
     # Custom builds
     - os: linux
```

Comment:
We should ping ILM about the fact that adding a Release config here breaks things. From what I read it could be a gcc compiler bug and they may want to look into how they set up library dependencies.